deployment/network-operator/README.md

```diff
@@ -0,0 +1,163 @@
+* Kubernetes secondary network for Network intensive workloads
+
+## QuickStart
+
+### Prerequisites
+
+- Kubernetes v1.15+
+- Helm v3
+- Ubuntu 18.04LTS
+
+
+### Install Helm
+
+Helm provides an install script to copy helm binary to your system:
+```
+$ curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3
+$ chmod 700 get_helm.sh
+$ ./get_helm.sh
+```
+
+For additional information and methods for installing Helm, refer to the official [helm website](https://helm.sh/)
+
+### Deploy Network Operator
+
+```
+# Add Repo
+$ helm repo add mellanox https://mellanox.github.io/network-operator
+$ helm repo update
+
+# Install Operator
+$ helm install -n network-operator --create-namespace --wait mellanox/network-operator network-operator
+
+# View deployed resources
+$ kubectl -n network-operator get pods
+```
+
+>__Note:__ By default the operator is deployed without an instance of `NicClusterPolicy`
+custom resource. The user is required to create it later with configuration matching the cluster or use
+chart parameters to deploy it together with the operator.
+
+## Chart parameters
+
+In order to tailor the deployment of the network operator to your cluster needs
+We have introduced the following Chart parameters.
+
+### General parameters
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `nfd.enabled` | bool | `True` | deploy Node Feature Discovery |
+| `operator.repository` | string | `mellanox` | Network Operator image repository |
+| `operator.image` | string | `network-operator` | Network Operator image name |
+| `operator.tag` | string | `None` | Network Operator image tag |
+| `deployCR` | bool | `false` | Deploy `NicClusterPolicy` custom resource according to provided parameters |
+
+### Custom resource parameters
+
+#### Mellanox OFED driver
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `ofedDriver.deploy` | bool | `false` | deploy Mellanox OFED driver container |
+| `ofedDriver.repository` | string | `mellanox` | Mellanox OFED driver image repository |
+| `ofedDriver.image` | string | `ofed-driver` | Mellanox OFED driver image name  |
+| `ofedDriver.version` | string | `5.0-2.1.8.0` | Mellanox OFED driver version  |
+
+#### NVIDIA Peer memory driver
+
+| Name | Type | Default | description |
+| ---- | ---- | ------- | ----------- |
+| `nvPeerDriver.deploy` | bool | `false` | deploy NVIDIA Peer memory driver container |
+| `nvPeerDriver.repository` | string | `mellanox` | NVIDIA Peer memory driver image repository |
+| `nvPeerDriver.image` | string | `nv-peer-mem-driver` | NVIDIA Peer memory driver image name  |
+| `nvPeerDriver.version` | string | `1.0-9` | Mellanox OFED driver version  |
+| `nvPeerDriver.gpuDriverSourcePath` | string | `/run/nvidia/driver` | GPU driver soruces root filesystem path(usually used in tandem with [gpu-operator](https://github.com/NVIDIA/gpu-operator)) |
```

Comment:
why this is config option. isn't this implementation details?

Reply:
the host path is determined by the gpu-operator so it may change.
in addition you may have the driver just installed on the host, in that case you would pass `/` here. 

in most cases you would not need to change it.